src/QMCWaveFunctions/tests/test_DiffOneBodyJastrowOrbital.cpp

```diff
@@ -0,0 +1,111 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2021 QMCPACK developers.
+//
+// File developed by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//
+// File created by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+#include "catch.hpp"
+
+#include "Particle/ParticleSetPool.h"
+#include "QMCWaveFunctions/Jastrow/DiffOneBodyJastrowOrbital.h"
+#include "QMCWaveFunctions/Jastrow/J1OrbitalSoA.h"
+#include "QMCWaveFunctions/Jastrow/RadialJastrowBuilder.h"
+#include "QMCWaveFunctions/WaveFunctionFactory.h"
+
+namespace qmcplusplus
+{
+TEST_CASE("J1 evaluate derivatives Jastrow", "[wavefunction]")
+{
+  Communicate* c;
+  c              = OHMMS::Controller;
+  auto ions_uptr = std::make_unique<ParticleSet>();
+  auto elec_uptr = std::make_unique<ParticleSet>();
+  ParticleSet& ions_(*ions_uptr);
+  ParticleSet& elec_(*elec_uptr);
+
+  ions_.setName("ion0");
+  ions_.create({1, 1});
```

Comment:
`create({1,1})` means two groups of particles, one in each group. If you need only 1 ion just do `create(1)`